sharded_queue/api.lua

```diff
@@ -319,24 +293,30 @@ function sharded_queue.statistics(tube_name)
             leader_only = true
         })
 
-    local ok, stats_collection = map('tube_statistic',
-        { tube_name = tube_name }, storages)
+    local stats_collection, err = cartridge_pool.map_call('tube_statistic',
```

Comment:
There is no such method in cartridge 1.2.0 - https://github.com/tarantool/cartridge/blob/1.2.0/cartridge/pool.lua

Please bump cartridge version in rockspec - https://github.com/tarantool/sharded-queue/blob/master/sharded-queue-scm-1.rockspec#L10

Reply:
Updated https://github.com/tarantool/sharded-queue/pull/19/commits/cbda209a89bcf79d4bd2096eeb7e643fe14b668d